corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -239,17 +241,17 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 $(e.target).is('a')                                                 // actual link, as in markdown
             )) {
                 e.preventDefault();
-                let model_id = this.model.get('id');
+                let modelId = this.model.get('id');
```

Comment:
What is the model id? Is it a backbone-generated id?

Reply:
It does come from formplayer. Not sure where it does get generated

Reply:
cool, I dug a little more and found it's the case id